include/mfmg/dealii/lanczos.templates.hpp

```diff
@@ -363,44 +316,25 @@ void Lanczos<OperatorType, VectorType>::details_calc_tridiag_epairs(
 template <typename OperatorType, typename VectorType>
 bool Lanczos<OperatorType, VectorType>::details_check_convergence(
     double beta, const int num_requested, double tol,
-    std::vector<double> const &evecs)
+    std::vector<double> const &evecs) const
 {
   const int n = evecs.size();
 
   // Must iterate at least until we have num_requested eigenpairs
   if (n < num_requested)
-  {
     return false;
-  }
 
   bool is_converged = true;
 
-  if (_verbosity > 0)
-  {
-    std::cout.precision(4);
-    std::cout << std::fixed;
-    // std::cout << "   " << beta;
-    std::cout << "   bounds ";
-  }
-
   // Terminate if every approximage eigenvalue has converged to tolerance
   // ISSUE: here ignoring possible nuances regarding the correctness
   // of this check.
   // NOTE: k may be desirable to "scale" the stopping criterion
   // based on (estimate of) matrix norm or similar.
   for (int i = 0; i < num_requested; ++i)
   {
-    const double bound = beta * abs(evecs[n - 1 + n * i]);
+    const double bound = beta * fabs(evecs[n - 1 + n * i]);
```

Comment:
They are `std::abs` overloads for floating-pointer numbers.  Why did you change?